--- a/drivers/gpu/drm/amd/pm/swsmu/smu11/sienna_cichlid_ppt.c
+++ b/drivers/gpu/drm/amd/pm/swsmu/smu11/sienna_cichlid_ppt.c
@@ -2576,10 +2576,10 @@
 static int sienna_cichlid_run_btc(struct smu_context *smu)
 {
 	int res;

 	res = smu_cmn_send_smc_msg(smu, SMU_MSG_RunDcBtc, NULL);
 	if (res)
-		dev_err(smu->adev->dev, "RunDcBtc failed!\n");
+		dev_warn(smu->adev->dev, "RunDcBtc failed, ignoring\n");

-	return res;
+	return 0;
 }
